platform/darwin/src/MGLRasterStyleLayer.h

```diff
@@ -7,16 +7,23 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- An `MGLRasterStyleLayer` is a style layer that renders raster tiles on the map.
+ An `MGLRasterStyleLayer` is a style layer that renders georeferenced raster
+ imagery on the map, especially raster tiles.
  
  Use a raster style layer to configure the color parameters of raster tiles
- loaded by an `MGLRasterSource` object. For example, you could use a raster
- style layer to render <a href="https://www.mapbox.com/satellite/">Mapbox
- Satellite</a> imagery, a <a
+ loaded by an `MGLRasterSource` object or raster images loaded by an
+ `MGLImageSource` object. For example, you could use a raster style layer to
+ render <a href="https://www.mapbox.com/satellite/">Mapbox Satellite</a>
+ imagery, a <a
  href="https://www.mapbox.com/help/define-tileset/#raster-tilesets">raster tile
  set</a> uploaded to Mapbox Studio, or a raster map authored in <a
  href="https://tilemill-project.github.io/tilemill/">TileMill</a>, the classic
  Mapbox Editor, or Mapbox Studio Classic.
+ 
+ Raster images may also be used as icons or patterns in a style layer. To
```

Comment:
If the intent is to direct them away to a different API for specific types of annotations/overlays I think we should also link to the docs for those (if possible), or leave out the specifics and say something to the effect of
> Use the Annotation APIs or Symbol layers with icons to ....

Reply:
In this SDK, we tend to refer to specific symbols whenever possible. jazzy automatically links references to API symbols, so the developer doesn’t have to do any searching to figure out what we’re referring to.